Gems/AtomLyIntegration/CommonFeatures/Code/Source/CoreLights/LightDelegateBase.h

```diff
@@ -82,12 +82,16 @@ namespace AZ
             // TransformNotificationBus::Handler overrides ...
             void OnTransformChanged(const Transform& local, const Transform& world) override;
 
+            LmbrCentral::ShapeComponentRequests* m_shapeBus = nullptr;
+
         private:
+            // Computes overall effect transform, taking shape translation offsets into account if applicable
+            AZ::Transform ComputeOverallTransform(const Transform& entityTransform);
```

Comment:
suggestion: Maybe have an overload of this with no parmeters that calls the TransformBus internally so the caller doesn't have to do that and pass it in (but if you have it cached you can use this overload which will be faster).